3_javascript/javascript1/week2/homework javascript1 week2/studentm.js

```diff
@@ -0,0 +1,32 @@
+//Student manager && addStudentToClass function
+
+let class07Students = [];
+function addStudentToClass(studentName) {
+  if (studentName === "") {
+    console.log("Kindly add more students to the class!");
+  }
+  for (let i = 0; i < class07Students.length; i++) {
```

Comment:
Excellent `for`-loop! 👍 You do not have to make this change, but can you think of a way to write this as a `for (... of ...)` instead? Which approach do you prefer of the two?

Reply:
Noted and Done